### PR TITLE
C350603 Searching POL by specifying acquisition method

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/e2e/orders/orders-export-to-a-vendor.cy.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ describe('orders: export', () => {
   const integartionDescription = 'Test Integation descripton1';
   const vendorEDICodeFor1Integration = getRandomPostfix();
   const libraryEDICodeFor1Integration = getRandomPostfix();
+  let user;
+  let firstPOLNumber;
```
Comment (MaksimTerekhov) [inline cypress/e2e/orders/orders-export-to-a-vendor.cy.js:40]: 'firstPOLNumber' is defined but never used.

File: cypress/e2e/orders/orders-export-to-a-vendor.cy.js [hunk 2]
```diff
@@ -36,6 +36,11 @@ describe('orders: export', () => {
...
+  let secondPOLNumber;
```
Comment (MaksimTerekhov) [inline cypress/e2e/orders/orders-export-to-a-vendor.cy.js:41]: 'secondPOLNumber' is defined but never used.

File: cypress/e2e/orders/orders-export-to-a-vendor.cy.js [hunk 3]
```diff
@@ -36,6 +36,11 @@ describe('orders: export', () => {
...
+  let thirdPOLNumber;
```
Comment (MaksimTerekhov) [inline cypress/e2e/orders/orders-export-to-a-vendor.cy.js:42]: 'thirdPOLNumber' is defined but never used.

File: cypress/support/fragments/orders/orderLines.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ import {
 import SearchHelper from '../finance/financeHelper';
 import getRandomPostfix from '../../utils/stringTools';
 import SelectInstanceModal from './selectInstanceModal';
+import { getLongDelay } from '../../utils/cypressTools';
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/orders/orderLines.js:24]: 'getLongDelay' is defined but never used.

**Review by MaksimTerekhov — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

File: cypress/e2e/orders/orders-export-to-a-vendor.cy.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/e2e/orders/orders-export-to-a-vendor.cy.js]: fixed

**Pull request merged**